test/test_util/test_trials_callback.py

```diff
@@ -56,7 +56,7 @@ def test_trials_callback_execution(self):
         cls = AutoSklearnClassifier(time_left_for_this_task=30,
                                     initial_configurations_via_metalearning=0,
                                     per_run_time_limit=10,
-                                    memory_limit=1024,
+                                    memory_limit=2048,
```

Comment:
Not sure why this was increased?

Reply:
I rerunned at with 1024 and got a memory out error